Dockerfile

```diff
@@ -44,7 +44,10 @@ RUN git clone https://github.com/ethereum-optimism/optimism.git && \
     git checkout develop && \
     git pull origin develop && \
     pnpm install && \
-    pnpm build
+    pnpm build && \
+    cd op-node && \
+    make
```

Comment:
whats this dockerfile here for again?

Reply:
to make the contract deployment

Op node generates the config for L2 genesis

Reply:
previously I was always compiling op node. We can probably copy the binary straight into the builder

Reply:
but we still need the solidity files to run the forge script. Not sure if we can forge a remote solidity file.

Reply:
yeah makes sense!